cloudinit/config/modules.py

```diff
@@ -129,6 +144,22 @@ def _read_modules(self, name):
         return module_list
 
     def _fixup_modules(self, raw_mods):
```

Comment:
Worth typing here to give concrete validation of return values? Maybe a pain for this at the moment.